lib/rubygems/resolver/installer_set.rb

```diff
@@ -193,7 +193,7 @@ def load_spec name, ver, platform, source # :nodoc:
   # Has a local gem for +dep_name+ been added to this set?
 
   def local? dep_name # :nodoc:
-    spec, = @local[dep_name]
+    spec = @local[dep_name]
```

Comment:
Thank you for your comment. Maybe you can point on some places in the rubygems that can be useful for the newbie contributor like me to help this gem out?